soci-snapshotter.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: soci-snapshotter
+  version: "0.9.0"
+  epoch: 0
+  description: A containerd snapshotter plugin which enables standard OCI images to be lazily loaded without requiring a build-time conversion step.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - containerd
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - go
+      - zlib-dev
+      - zlib-static
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/awslabs/soci-snapshotter
+      tag: v${{package.version}}
+      expected-commit: 737f61a3db40c386f997c1f126344158aa3ad43c
+
+  - runs: |
+      make
+      make CMD_DESTDIR=${{targets.contextdir}} install
+
+  - uses: strip
+
+subpackages:
+  - name: soci-snapshotter-static
+    dependencies:
+      runtime:
+        - soci-snapshotter
+    options:
+      # Do not provide any embedded components
+      no-provides: true
```

Comment:
What embedded components are we filtering out with this?

Reply:
providing zlib: https://github.com/awslabs/soci-snapshotter/blob/main/docs/build.md#dependencies